src/assets/src/components/queue.tsx

```diff
@@ -450,6 +458,8 @@ export function QueuePage(props: PageProps<QueuePageParams>) {
     const [myUser, setMyUser] = useState(undefined as MyUser | undefined);
     const userWebSocketError = useUserWebSocket(props.user!.id, (u) => setMyUser(u as MyUser));
     const [showMeetingTypeDialog, setShowMeetingTypeDialog] = useState(false);
+    const [dialogState, setStateAndOpenDialog] = useDialogState();
```

Comment:
I don't have further thoughts on this, what you did seems to be rational. I haven't been coding from React side since CCM to share additional thoughts. The change seems to be functional.